CHANGELOG.md

```diff
@@ -4,6 +4,38 @@ All notable changes to this project will be documented in this file.
 
 The format follows [keepachangelog.com]. Please stick to it.
 
+## [2.6.2 Toothless Taipan] -- unreleased
+
+### Added
+
+* New checksum types metro and highway
+* Additional unit tests
+* New option --keep-hardlinked
+* --dedupe option can deduplicate twins on any reflick-capable filesystem
+* --dedupe-readonly option can dedupe files on read-only btrfs snapshots
+
+### Changed
+
+* Checksum types for -P... options (see https://github.com/sahib/rmlint/issues/261)
+* Under-the-hood changes for checksum.c
```

Comment:
Since the changelog is something that's supposed to be user faces, I'd not mention things that do not have any effect for users. (Same as for additional unit tests)